cypress/e2e/po/components/sortable-table.po.ts

```diff
@@ -252,4 +252,8 @@ export default class SortableTablePo extends ComponentPo {
   pagination() {
     return new PaginationPo();
   }
+
+  paginationGlobalUsersRoles() {
+    return new PaginationPo('#GLOBAL div.paging');
+  }
```

Comment:
this is specific to `cypress/e2e/po/pages/users-and-auth/roles.po.ts` so should live there and should account for other tabs

Reply:
I'll take a look